cloud/deploys/staging_azure/main.tf

```diff
@@ -19,15 +19,23 @@ module "app" {
 
   postgres_admin_login    = var.postgres_admin_login
   postgres_admin_password = var.postgres_admin_password
+  # note that we must use GP tier
+  postgres_sku_name       = "GP_Gen5_2"
```

Comment:
question (blocking): should this use the variable/its default rather than be harded code?

Reply:
ahh it is one but for staging we def want to use this tier